src/drive/web/modules/drive/Toolbar/components/NewModal.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+
+import Modal, {
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalButtons
+} from 'cozy-ui/react/Modal'
+import { Button, Icon } from 'cozy-ui/react'
+import { translate } from 'cozy-ui/react/I18n'
+import withBreakpoints from 'cozy-ui/react/helpers/withBreakpoints'
+import classNames from 'classnames'
+import styles from './styles.styl'
+
+class NewModal extends Component {
+  render() {
+    const {
+      title,
+      dismissAction,
+      primaryAction,
+      primaryText,
+      t,
+      secondaryText,
+      secondaryAction,
+      description,
+      breakpoints: { isMobile }
+    } = this.props
+    //!TODO Move this new kind of Modal to UI
+    return (
+      <Modal
+        mobileFullscreen
+        closable={isMobile ? false : true}
+        size="large"
+        title={isMobile ? '' : title}
+        dismissAction={dismissAction}
+      >
+        {isMobile && (
+          <ModalHeader className={classNames(styles['modal-header'])}>
+            <h2>{title}</h2>
```

Comment:
I'm surprised we can use a tag like this and it actually renders what we want? Isn't there a component in cozy-ui that fits the requirements?

Reply:
In fact... https://github.com/cozy/cozy-ui/blob/master/react/Modal/ModalHeader.jsx#L23

As I wanted to be as close as possible to the original to put this component to UI ;)